site/content/en/docs/manual/advanced/xml_format.md

```diff
@@ -41,6 +41,7 @@ In annotation mode each image tag has `width` and `height` attributes for the sa
           <attributes>
             <attribute>
               <name>String: attribute name</name>
+              <type>String: any, bbox, cuboid, cuboid_3d, ellipse, polygon, polyline, points, skeleton, tag</type>
```

Comment:
Is it for attributes? I believe it is a type for the label itself.

Reply:
Fixed, thanks